recipe/meta.yaml

```
@@ -12,34 +12,38 @@ source:
   sha256: 4295085f11221c54b1dd2b1d39a675a85dfd9f900294297567e1d36f65ac4841
 
 build:
-  number: 1
+  number: 2
   rpaths:
     - lib/R/lib/
     - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
 
 requirements:
   build:
-    - {{ compiler('c') }}              # [not win]
-    - {{ stdlib("c") }}                # [not win]
-    - {{ compiler('m2w64_c') }}        # [win]
-    - {{ stdlib("m2w64_c") }}          # [win]
-    - {{ compiler('cxx') }}            # [not win]
-    - {{ compiler('m2w64_cxx') }}      # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+    - {{ compiler('c') }}          # [not win]
+    - {{ stdlib("c") }}            # [not win]
+    - {{ compiler('m2w64_c') }}    # [win]
+    - {{ stdlib("m2w64_c") }}      # [win]
+    - {{ compiler('cxx') }}        # [not win]
+    - {{ compiler('m2w64_cxx') }}  # [win]
     - {{ posix }}filesystem        # [win]
     - {{ posix }}make
     - {{ posix }}sed               # [win]
     - {{ posix }}coreutils         # [win]
     - {{ posix }}zip               # [win]
-    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
   host:
     - r-base
   run:
     - r-base
 
 test:
   commands:
-    - $R -e "library('cli')"           # [not win]
-    - "\"%R%\" -e \"library('cli')\""  # [win]
+    - $R -e "library('cli')"                  # [not win]
+    - "\"%R%\" -e \"library('cli')\""         # [win]
```

That said, am thinking we can do something like this, which keeps this behavior limited to the package tests

```suggestion
    - export CLI_NO_THREAD=1                  # [not win]
    - set CLI_NO_THREAD=1                     # [win]
    - $R -e "library('cli')"                  # [not win]
    - "\"%R%\" -e \"library('cli')\""         # [win]
```

---

We considered this, but it requires setting this on every feedstock that builds ppc64le and depends on this package.

---

Ah ok. Did not appreciate the breadth of the impact

In that case let's check for `CONDA_BUILD` in the activation script